pkg/tsdb/tsdbtest/config_test.go

```diff
@@ -60,9 +68,9 @@ func TestGetV3ioConfigPath(t *testing.T) {
 		{description: "get config from env var",
 			expectedPath: config.DefaultConfigurationFileName,
 			setup: func() func() {
-				os.Setenv(V3ioConfigEnvironmentVariable, config.DefaultConfigurationFileName)
+				os.Setenv(config.V3ioConfigEnvironmentVariable, config.DefaultConfigurationFileName)
```

Comment:
Here you are overriding the "maybe existing" config path, dont think you want to do that.

Reply:
It just recovers it's original value. In case I run the following command:
`V3IO_CONF=<real-path>/v3io.yaml go test -tags "integration unit" ../../...`
This test should ignore the env but recover the state afterwards